src/main/resources/ListedPrintSerial.json

```diff
@@ -107,6 +107,22 @@
             "default": "Main"
           }
         },
+        {
+          "field": "PO_LINE_ESTIMATED_PRICE",
```

Comment:
I think we're now calculating this field in mod-orders, so whatever you have here will probably be overridden...

Reply:
yea it will, but we were already calculating it, so just left the field here. I think there is no harm in calculating it here as well?

Reply:
My take here will is maybe we shouldn't calculate it here if it is going to be overridden in mod-orders. It's just extra code ...

Reply:
Removed the code